lib/Pool.js

```diff
@@ -17,9 +17,10 @@ function Pool(options) {
   this._freeConnections      = [];
   this._connectionQueue      = [];
   this._closed               = false;
+  this._pendingClosing       = false;
 }
 
-Pool.prototype.getConnection = function (cb) {
+Pool.prototype.getConnection = function (cb, queued) {
```

Comment:
Please add documentation for the new queued argument.

Reply:
This queued argument is a private argument, is it appropriate to add documentation to README.md?

Reply:
There is no such thing as a private argument -- the user can simply pass in the value. Just not documenting it doesn't make it private. If the argument is there, it needs to be documented.